src/app/shared/date/datepicker.component.ts

```diff
@@ -1,72 +0,0 @@
-import { Component, Input, Output, EventEmitter, AfterViewInit, ViewChild, ElementRef } from '@angular/core';
-import * as Pikaday from 'pikaday';
```

Comment:
I _think_ you can also clean pikaday out of `package.json` now, right?  But not moment, since that's used outside of the datepicker.